extern/ProtoGeometry/ProtoGeometry.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <ProtoGeometryConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <GeometryFactoryFileName>libg_229_0_0/LibG.ProtoInterface.dll</GeometryFactoryFileName>
+  <GeometryFactoryFileName>libg_230_0_0/LibG.ProtoInterface.dll</GeometryFactoryFileName>
```

Comment:
did this cause test failures?

Reply:
I'm not even sure if this is still being used. From what I saw running workflowtests locally it didn't seem to be using it.